### DISCUSSION TITLE
How do I match extension param on route?

### DISCUSSION THREAD

**Discussion by shtse8** [nitrojs/nitro#372] [Q&A]
I tried `/server/routes/favicon.[format].ts` and  `/server/routes/favicon.[format].get.ts`, but it doesn't work.
How can I match a extension param (which after a dot)?

**Comment by nWacky** [comment]
With nitro 2.9.7 I managed to add an extension by specifying an extension and a method in the url.

Like this:

- `routes/api/favicon.json.get.ts`
- `routes/api/favicon.json5.get.ts`

In the future versions of nitro, mixed params in the same path segment should be supported.
It was added it [feat!: support mixed params in same path segment (rou3/#52)](https://github.com/unjs/rou3/pull/52)

Mixed params will not be supported in radix3, but will be supported in rou3 (next version of radix3). Issue: ([rou/#148](https://github.com/unjs/rou3/issues/148))
